src/server/config/env/runTime.js

```diff
@@ -78,11 +88,49 @@ const runTime = [
     },
     defaultValue: () => 3005,
   },
-  // holocron config, the modules and versions to use
+  {
+    name: 'HTTP_ONE_APP_DEV_CDN_PORT',
```

Comment:
Added some of these env vars that were not previously added to this file for consistency. As currently implemented there is default env var value logic sprinkled throughout the code and so with this we can centralize that logic.